### PR TITLE
WIP: Translation integration

### PR DESCRIPTION
**Pull request by izwick-schachter**
I totally integrated translations into broadcast. Now, there is no longer text+translation stored in the broadcast item record, it's all in a translation record. This will be good to merge as soon as #79 is merged and a migration is written that will cleanly and reversibly move the data from the broadcasts table to the translation table. I took a stab at it, but it needs to be looked over.

### CODE REVIEW

**Comment by izwick-schachter** [comment]
@ArtOfCode- Ideas for a better migration? That's the bit that really needs work.

File: app/controllers/broadcast/items_controller.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Broadcast::ItemsController < ApplicationController
   before_action :set_item, except: [:index, :new, :create, :setup_generation, :generate_script, :need_translation, :scripts, :view_script]
 
   def index
+    @language = Translations::Language[params[:lang] || 'en-US']
     @items = conditional_filter Broadcast::Item.active, originated_at: params[:originated_at], broadcast_municipality_id: params[:municipality],
                                                         source: params[:source], id: params[:id]
     @items = @items.includes(:municipality).order(originated_at: :desc).paginate page: params[:page], per_page: 100
```
Comment (ArtOfCode-) [inline app/controllers/broadcast/items_controller.rb]: Call `includes(association_name)` to do a soft join without actually needing to do any filtering on the joined table.

`@items.includes(:municipality).includes(:translations)...`

Comment (izwick-schachter) [reply app/controllers/broadcast/items_controller.rb]: This appears to not have made much difference in the queries I see in the logs.

**Comment by izwick-schachter** [comment]
@ArtOfCode- OK, addressed those changes. Got more changes or want to talk migration? I'd like to merge at some point tomorrow.

**Pull request merged**